docs/markdown/Quick-guide.md

```diff
@@ -35,11 +35,11 @@ generate native VS and Xcode project files.*
 Installation using package manager
 --
 
-Ubuntu:
+Ubuntu/Debian:
 
 ```console
 $ sudo apt-get install python3 python3-pip python3-setuptools \
-                       python3-wheel ninja-build
+                       python3-wheel ninja-build python3-mesonpy
```

Comment:
python3-mesonpy is not meson.

Reply:
Specifically, mesonpy is a build backend for python wheels that uses meson; it is maintained in a separate repository as https://github.com/mesonbuild/meson-python